locale/ca/about/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: about.hbs
+title: Sobre nosaltres
+trademark: Trademark
+---
+# Sobre Node.js&reg;
+
+Concebit com a un entorne d' execució de JavaScript orientat a events asíncrons, Node està dissenyat
+per construir aplicacions en xarxa escalables. En la seguent aplicació de exemple "hola món", es pot
+manegar moltes conexions concurrents. Per cada conexió el *callback* serà executat, no obstant això
+si no hi ha treball que fer Node estarà dormint.
+
+```javascript
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hola Món\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`El servidor s'està executant en http://${hostname}:${port}/`);
+});
+```
+Això contrasta amb el model de concurrència més comú avui dia, on s'usen fils del
+Sistema Operatiu. Les operacions de xarxes basades en fils són relativament ineficients
+i són molt difícils d'usar. A més, els usuaris de Node són lliures de preocupacions
+sobre el bloqueig del procés, ja que no existeix. Gairebé cap funció en Node realitza
+I/O directament, així que el procés mai es bloqueja. A causa de que no hi ha bloqueig
+és molt raonable desenvolupar sistemes escalables en Node.
+
+Si algú d'aquests termes no li és familiar, hi ha un article complet en
```

Comment:
Si cap d'aquests termes no li és familiar, hi ha un article complet en

Reply:
Also done! Thx for reviewing it ;)